src/Tribe/Views/V2/View.php

```diff
@@ -1498,7 +1499,7 @@ static function () {
 			'before_events'        => tribe( Advanced_Display::class )->get_before_events_html( $this ),
 			'after_events'         => tribe( Advanced_Display::class )->get_after_events_html( $this ),
 			'display_events_bar'   => $this->filter_display_events_bar( $this->display_events_bar ),
-			'disable_event_search' => tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ),
+			'disable_event_search' => Admin_Helpers::instance()->is_screen( 'tribe-admin-manager' ) ? false : tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ),
```

Comment:
Same exact problem as in the other PR, but in this case we already have the filters so that you can modify just for the manager instead of adding the overhead here.

Reply:
@bordoni Question regarding this approach: Since the Events Manager is a ECP specific feature ( and that's where the filter lives ) what's the best way to include it in TEC?

As things stand, it works as expected when both TEC and ECP are active but immediately I deactivate ECP things stop working.